rts/System/SpringApp.h

```diff
@@ -62,6 +64,9 @@ class SpringApp
 	// this gets passed along to PreGame (or SelectMenu then PreGame),
 	// and from thereon to GameServer if this client is also the host
 	std::shared_ptr<ClientSetup> clientSetup;
+
+	// used to delay relaying messages to console to avoid issues
```

Comment:
what issues, and how does delay help?

Reply:
It was because we don't want diagnostic messages appearing in console output if the user is issuing a command where the output will be used as input for another process. Badosu had this issue, I think with generating weapon defs - the cpu information broke the output format. Alternatively, I could look to moving the messages; though, this current solution provides some protection should other kinds of loggin be added in the future prior to the Spring App starting.